modules/admin_manual/pages/configuration/server/caching_configuration.adoc

```diff
@@ -141,12 +141,12 @@ of what to look for:
 [source,php,subs="attributes+"]
 ----
 'redis' => [
-    'host' => 'localhost',  // Can also be a unix domain socket => '/tmp/redis.sock'
-    'port' => {std-port-redis},
+    'host' => 'localhost',       // For a unix domain socket => '/var/run/redis/redis.sock'
```

Comment:
```suggestion
    'host' => 'localhost',       // For a unix domain socket 'host ' => '/var/run/redis/redis.sock'
```
Maybe this will make it clearer that the key stays as `host` and the value should be `/var/run/redis/redis.sock`